### PR TITLE
Add base schema to the blocks

### PR DESCRIPTION
**Pull request by Klynger**
#### How to test it? \*
[Link to see the schema](https://klynger--storecomponents.myvtex.com/admin/cms/site-editor)

To test the `target` prop follow the instructions

1. Open the [workspace](https://klynger--storecomponents.myvtex.com/)
2. Open the quickview
3. Click in `Mais detalhes >`

It should open the product page in another tab

### CODE REVIEW

File: react/StoreLink.tsx [hunk 1]
```diff
@@ -24,10 +35,25 @@ export default function StoreLink(props: Props) {
     setShouldReplaceUrl(Boolean(modalDispatch))
   }, [modalDispatch])
 
-  return (
+  return target && target !== '_self' ? (
+    <a target={target} href={href} className={handles.link}>
```
Comment (klzns) [inline react/StoreLink.tsx]: Consider fixing Link component instead

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,12 +7,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
```
Comment (victorhmp) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,12 +7,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- Schema to the blocks.
+- `target` prop to all links.
+
 ## [0.1.1] - 2020-02-13
 
 ### Fixed
-- Improper section for props was removed
+
```
Comment (victorhmp) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,12 +7,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- Improper section for props was removed.
 
 ## [0.1.0] - 2020-01-31
 
 ### Added
+
```
Comment (victorhmp) [inline CHANGELOG.md]:
```suggestion
```

File: messages/pt.json [hunk 5]
```diff
@@ -1,4 +1,6 @@
 {
-  "admin/test": "teste",
-  "store/test": "teste"
-}
\ No newline at end of file
+  "admin/editor.link.label.title": "Texto do link",
+  "admin/editor.link.label.description": "Qualquer text ex. \"Categorias\", \"Sobre nós\", etc.",
```
Comment (victorhmp) [inline messages/pt.json]:
```suggestion
  "admin/editor.link.label.description": "Qualquer texto ex. \"Categorias\", \"Sobre nós\", etc.",
```

File: messages/pt.json [hunk 6]
```diff
@@ -1,4 +1,6 @@
...
+  "admin/editor.link.href.description": "Link de fato ex. \"https://www.google.com\", \"/sobre-nos\", etc",
```
Comment (victorhmp) [inline messages/pt.json]: Maybe a better message would be:
`"URL para o link ex. \"https://www.google.com\", \"/sobre-nos\", etc"`

File: react/ProductLink.tsx [hunk 7]
```diff
@@ -1,9 +1,10 @@
+import React, { useState, useEffect } from 'react'
```
Comment (victorhmp) [inline react/ProductLink.tsx:1]: 🙏

File: react/ProductLink.tsx [hunk 8]
```diff
@@ -43,10 +45,23 @@ export default function ProductLink(props: Props) {
...
+    </a>
+  ) : (
     <Link to={resolvedLink} className={handles.link} replace={shouldReplaceUrl}>
       {label && <span className={handles.label}>{label}</span>}
-      {children && <div className={handles.childrenContainer}>{children}</div>}
+      {hasChildren(children) && (
+        <div className={handles.childrenContainer}>{children}</div>
+      )}
     </Link>
   )
```
Comment (victorhmp) [inline react/ProductLink.tsx]: I guess this logic is not needed anymore after https://github.com/vtex-apps/render-runtime/pull/486

File: react/StoreLink.tsx [hunk 9]
```diff
@@ -24,10 +35,25 @@ export default function StoreLink(props: Props) {
...
+      {hasChildren(children) && (
+        <div className={handles.childrenContainer}>{children}</div>
+      )}
     </Link>
   )
 }
+
+StoreLink.schema = {
+  title: 'Link',
+}
```
Comment (victorhmp) [inline react/StoreLink.tsx:55]: Don't forget to do this using `messages`

**Review by victorhmp — CHANGES_REQUESTED** [review]

File: react/StoreLink.tsx (hunk 9 above)
Comment (Klynger) [reply react/StoreLink.tsx:55]: I think I shouldn't translate `Link`

**Review by klzns — APPROVED** [review]

**Review by victorhmp — APPROVED** [review]

**Pull request merged**